index.html

```diff
@@ -50,6 +50,212 @@ <h3 class="box3">
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot3.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Multi-Post
+            Stories
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="proj-myself">
+        <h1 class="my-title">
+            About Myself
+        </h1>
+        <p class="box2">
+            I'm a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h3 class="box3">
+            LET'S CONNECT
+        </h3>
+        <ul class="box4">
+            <li class="social-media">
+                <img src="images/twitter.png" alt="twitter">
+            </li>
+            <li class="social-media">
+                <img src="images/Linkedin.png" alt="Linkedln">
+            </li>
+            <li class="social-media">
+                <img src="images/m.png" alt="m">
+            </li>
+            <li class="social-media">
+                <img src="images/github.png" alt="github">
+            </li>
+            <li class="social-media">
+                <img src="images/fingers.png" alt="two fingers">
+            </li>
+        </ul>
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
```

Comment:
Please change the button content to "Get my resume" according to the requirement.